app/src/screens/Onboarding/SignInScreen.tsx

```diff
@@ -14,7 +35,41 @@ type SignInScreenProps = StackScreenProps<OnboardingStackParams, 'SignInScreen'>
 
 export const SignInScreen = ({ navigation }: SignInScreenProps) => {
   return (
-    <FullScreenWhite>
+    <SignInMainContainer>
```

Comment:
Replaced FullScreen view with this main container, which branches off from FullScreen and adds page padding